mini-cap/src/pages/CondoDetails.jsx

```diff
@@ -62,139 +104,142 @@ export default function CondoDetails({ name, address, parkingCount, lockerCount,
 						<div className="condo-info">
 							<div className='name-and-number'>
 								<div className= 'pic-and-num'>
-									{<img src="https://ksquarecondos.com/wp-content/uploads/2019/09/q7l25-ts-c80_Gro-.%E5%89%AF%E6%9C%AC.jpg" alt="Profile" className="profile-picture" />}
+									{picture && <img src={picture} alt="Profile" className="profile-picture" />}
 								</div>
 								<div className='pic-and-tag'>
-								{condoStatus === "vacant" && role === 'company' && ( 
+								{status === "Vacant" && role === 'mgmt' && (
+									<>
+									<div>
+										<div className={`user-tag vacant`}>{status}</div>
+										{role === "mgmt" && (
```

Comment:
use the RENTER_OWNER and MANAGEMENT_COMPANY constants from Constants.js in the checks instead of "mgmt" and "renter/owner"